### ISSUE TITLE
jest testing when requiring static assets

### ISSUE THREAD

**Issue by raythree** [coryhouse/react-slingshot#456]
I found an issue with jest tests in the current slingshot version that is easily reproducible as follows:

```
1. Add a PNG like src/images/logo.png
2. Edit FuelSavingsForm by adding the following after the imports:

const logo = require('../images/eissbox-logo-sm.png');

3. Reference it in the render method:

        <h2>Fuel Savings Analysis</h2>
        <img src={logo} />
        <table>
```

If you run "npm start -s" you'll see the logo as expected.

Now, run "npm run test" an you will get the following failure:

```
src/components/FuelSavingsForm.spec.js
  Test suite failed to run

    /Users/bill/projects/react-slingshot/src/images/eissbox-logo-sm.png:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){�PNG
                                                                                             ^
    SyntaxError: Invalid or unexpected token

      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/ScriptTransformer.js:289:17)
      at Object.<anonymous> (src/components/FuelSavingsForm.js:6:12)
      at Object.<anonymous> (src/components/FuelSavingsForm.spec.js:3:24)
```
I'm sure it's related to this:

https://github.com/facebook/jest/issues/2663
 
but have not yet resolves it.

**Comment by coryhouse** [comment]
Good catch. The last comment on this thread should resolve this: https://github.com/facebook/jest/issues/2663#issuecomment-317109798

That's the same approach used by create-react-app.

Would you be willing to submit a PR if that fixes your issue?

**Comment by raythree** [comment]
Sure, I'll give it a try, thanks.

**Comment by kwelch** [comment]
This should be resolved with #457, correct?
